components/VenueMapCard.tsx

```diff
@@ -37,32 +37,21 @@ export default function VenueMapCard({
                     {venue.name}
                   </a>
                 </p>
-                <a href={`venues/${venue.id}`} className="mt-2 block">
-                  <p className="mt-3 text-primary text-lg">Address</p>
-                  <p className="text-primary text-sm">
-                    {venue.address.address_1}
-                  </p>
-                  <p className="text-primary text-sm">
-                    {venue.address?.address_2}
-                  </p>
-                  <p className="text-primary text-sm">
-                    {venue.address.town_city}
-                  </p>
-                  <p className="text-primary text-sm">
-                    {venue.address.postcode}
-                  </p>
-                  <p className="text-primary text-sm">
-                    {venue.address.country}
-                  </p>
-                </a>
+              </div>
+              <div className="grid grid-cols-2 gap-2">
+                {venue?.attributes?.map((attribute) => {
```

Comment:
if you do a check for the venue as part of the return you won't need the question marks. I.e. 

```javascript
vneue && <div>
```